### PR TITLE
Coverage for testcsv.js

### PR DESCRIPTION
**Pull request by lizziegooding**
### Context

Adds tests for `lib/testcsv.js`. Goal is to test how pt2itp is handling certain known edge cases using synthetic data.

### Summary of Changes
- [x] added test fixtures for Rhode Island addresses
- [x] fixed a bug in `lib/testcsv.js` that prevented csv files with less than 25 or less than a multiple of 25 lines from being geocoded
- [x] fixed typo in `lib/geocode.js`
- [x] added tests tracking `stdout` and errors written to output csv in `test/testcsv.js` using fastcsv and tape-spawn
- [x] removed .mbtiles files from the gitignore

### Next Steps
- [ ] preliminary review from @ingalls or @yhahn

### CODE REVIEW

File: test/testcsv.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    let index = `${__dirname}/fixtures/index-ri-single/us_ri-address-both-0d603c2a171017011038-0d603c2a39.mbtiles`;
+    let input = `${__dirname}/fixtures/index-ri-testing.csv`;
+    let output = `${__dirname}/fixtures/index-ri-errors.csv`;
+    let config = `${__dirname}/fixtures/index-ri-carmen-config.json`;
+
+    t.test('Return correct std.err message', (t) => {
+
+        let st = spawn(t, `${__dirname}/../index.js testcsv --index ${index} --input ${input} --output ${output} --config ${config}`);
+        st.stderr.match('\nERROR TYPE                   COUNT\n-----------------------------------------------------------------------------------\nDIST                             2 ( 50.0% of errors | 18.2% of total addresses)\nNO RESULTS                       2 ( 50.0% of errors | 18.2% of total addresses)\n\nok - 4/11 (36.4%) failed to geocode\n');
```
Comment (yhahn) [inline test/testcsv.test.js]: Would it be possible to write/capture this as a JS templated multiline string to make it easier to read? e.g.

```
st.stderr.match(`
ERROR TYPE                   COUNT
-----------------------------------------------------------------------------------
DIST                             2 ( 50.0% of errors | 18.2% of total addresses)
```

etc.

File: lib/geocode.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ function localCarmen(param) {
     if (param.getInfo.metadata) param.getInfo = param.getInfo.metadata; //Necessary for internal use
 
     delete param.getInfo.tiles;
-    delete param.getInfo.geocdoer_data;
+    delete param.getInfo.geocoder_data;
```
Comment (yhahn) [inline lib/geocode.js]: :joy:

File: package.json [hunk 3]
```diff
@@ -39,6 +39,7 @@
     "simple-statistics": "^5.0.0",
     "split": "^1.0.0",
     "talisman": "^0.20.0",
+    "tape-spawn": "^1.4.2",
```
Comment (ingalls) [inline package.json]: Can we move this to `devDependencies`? Shouldn't be necessary for a prod run.

**Comment by mattficke** [comment]
```
Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.21' not found
```
@lizziegooding saw you were running into this error, I think you may need to `apt-get install libstdc++-5-dev` in the circle config. If you rebuild with ssh, you can see the installed GLIBCXX versions:

```
$ strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_DEBUG_MESSAGE_LENGTH
```
And the installed libstdc++ version:

```
$ apt-cache policy libstdc++ | grep -B 6 100
libstdc++-4.9-dev:
  Installed: 4.9.2-10
  Candidate: 4.9.2-10
  Version table:
 *** 4.9.2-10 0
        500 http://deb.debian.org/debian/ jessie/main amd64 Packages
        100 /var/lib/dpkg/status
--
libstdc++6:
  Installed: 4.9.2-10
  Candidate: 4.9.2-10
  Version table:
 *** 4.9.2-10 0
        500 http://deb.debian.org/debian/ jessie/main amd64 Packages
        100 /var/lib/dpkg/status
```
